BDArmory/Weapons/Missiles/BDModularGuidance.cs

```diff
@@ -1577,6 +1577,28 @@ public override Vector3 GetForwardTransform()
             return GetTransform(ForwardTransformAxis);
         }
 
+        public override float GetKinematicTime()
+        {
+            if (!_missileIgnited) return -1f;
+
+            float deltaV = (float)vessel.GetDeltaV();
+            float accel = VesselModuleRegistry.GetModuleEngines(weaponManager.incomingMissileVessel).Where(e => e != null && e.EngineIgnited && e.isOperational).Sum(e => e.GetCurrentThrust()) / weaponManager.incomingMissileVessel.GetTotalMass();
+            float missileKinematicTime = deltaV / accel; // FIXME, there must be a better way to calculate burn time
+            float drag = vessel.parts.Sum(x => x.dragScalar);
```

Comment:
The drag part of this calculation ought to be skipped if the missile is in a vacuum.